kapre/backend.py

```diff
@@ -210,3 +210,54 @@ def mu_law_decoding(signal_mu, quantization_channels):
         tf.math.sign(signal) * (tf.math.exp(tf.math.abs(signal) * tf.math.log1p(mu)) - 1.0) / mu
     )
     return signal
+
+
+def freq_mask(self, input_f, param, name='freq_mask'):
```

Comment:
(** no `self` in a function).

I think `freq_mask` and `time_mask` should be merged into one universal function. Maybe with a name like `random_masking_along_axis` or something, so..

```python
def random_masking_along_axis(x, axis, x_min=0, x_max=None, max_width=None): 
  if x_max is None:
    x_max = x.shape[axis]
  if max_width is None:
    max_width = (x_max - x_min) // 4  # 4 is an arbitrary choice though
  # do the work..
```

Reply:
Sorry, typo error when I was moving to backend, it was part of my class, Im trying to follow new Kapre but it has changed a lot